tests/reporter/kubernetes.go

```diff
@@ -24,6 +24,14 @@ import (
 	"kubevirt.io/kubevirt/tests"
 )
 
+const (
+	sriovEntityURITemplate       = "/apis/sriovnetwork.openshift.io/v1/namespaces/%s/%s/"
+	sriovNetworks                = "sriovnetworks"
```

Comment:
Is not this "entity" too? One would like to use `sriovNetworks` as a variable name, maybe we should add a suffix

Reply:
I don't understand what you mean, nor the motivation behind it.

Reply:
You define following constants: sriovNetworks sriovNodeNetworkPolicyEntity sriovNodeStateEntity sriovOperatorConfigs

Half of them has "Entity" suffix, the other does not. They are all names of types, right? So I would expect the nameing to be consistent.

The second part of my comment is just complaining that you waste `sriovNetworks`  as a constant name. What do I do then when I want to list all sriov networks? Call the variable `sriovNetworkList`? I think it would be better to call the constant more specifically `sriovNetworksType` or `Entity` if you want. The constant does not contain any sriov network after all.

Reply:
Done.